### PR TITLE
[explorer]: add Banner variances

### PR DESCRIPTION
**Pull request by plam-ml**
## Description 

https://mysten.atlassian.net/browse/APPS-730
- Add more variances to banner

Screenshot 2023-04-04 at 1 40 55 PM

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/explorer/src/ui/Banner.tsx [hunk 1]
```diff
@@ -29,27 +33,70 @@ const bannerStyles = cva(
...
+            border: {
+                true: '',
+            },
         },
         defaultVariants: {
             variant: 'message',
             spacing: 'md',
+            border: false,
         },
+        compoundVariants: [
```
Comment (plam-ml) [inline apps/explorer/src/ui/Banner.tsx:48]: This is a bit customized, but I do not think we have all the exact hex values for these colors

Comment (Jordan-Mysten) [reply apps/explorer/src/ui/Banner.tsx:47]: Can you work with @mystie711 to get these incorporated into the theme?

**Review by mystie711 — APPROVED** [review]
Brilliant!

File: apps/explorer/src/ui/Banner.tsx [hunk 2]
```diff
@@ -29,27 +33,69 @@ const bannerStyles = cva(
                 md: 'px-3 py-2',
                 lg: 'p-5',
             },
+            shadow: {
+                true: 'shadow-md',
+            },
+            border: {
```
Comment (pchrysochoidis) [inline apps/explorer/src/ui/Banner.tsx:39]: Not sure if this is necessary. I thought border is 'controlled' from variant, do we want to have toasts with no borders but being positive for example? Same for shadow I think

Comment (plam-ml) [reply apps/explorer/src/ui/Banner.tsx:39]: @pchrysochoidis I am going off from Figma reqs here

```
Toast / Message Banner
variables controllable via code switches

1-line or 2-line
Icon ON/OFF
Color good, warning, issue, neutral_grey, neutral_white, neutral_sui_blue_light, 
Stroke ON/OFF
Shadow ON/OFF
```

Both the `border` and `shadow` are toggle-able.

Comment (pchrysochoidis) [reply apps/explorer/src/ui/Banner.tsx:39]: I see, I would ask @mystie711 about this just to avoid doing all the compoundVariants stuff.

Comment (plam-ml) [reply apps/explorer/src/ui/Banner.tsx:39]: Yea, let me ping in our shared chat

**Pull request merged**